certificate-manager/byo-root.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: "Smallstep Certificate Manager: Bring Your Own Root"
+html_title: "Smallstep Certificate Manager: Bring Your Own Root"
+description: Use your existing root to sign an intermediate certificate for use in Smallstep
+---
+
+
+With the Smallstep platform, you can bring your existing PKI to us, by signing a Smallstep intermediate CA from your existing root.
+
+Intermediate CAs (also called subordinate CAs) are used to sign and issue leaf certificates to subscribers. Intermediates aren't generally included in trust stores, making them easier to revoke and rotate. This is why they are commonly used for online CAs like Smallstep Certificate Manager.
+
+This tutorial will walk you through importing your root CA certificate to Smallstep, and signing an intermediate CA that we'll use to issue certificates on your behalf.
+
+The steps are:
+1. Upload your root CA certificate to Smallstep.
+2. Customize your intermediate CA certificate properties. Smallstep will create an intermediate private key and a Certificate Signing Request (CSR) for you.
+3. Download the CSR and use your root CA to sign in
+4. Upload the signed certificate to Smallstep, and we will finish creating your Authority
+
+## Requirements
+
+All you need is a [Smallstep Certificate Manager](https://smallstep.com/signup) account and access to your existing root CA.
+This feature requires an Advanced Authority.
+It is recommended to use an airgapped machine to sign the CSR.
+
+## Overview
+
+The Smallstep app has a workflow for bringing your own root.
+To find it,
+
+1. Sign into your Smallstep team.
+2. Then, choose **Authorities** under **Certificate Manager** in the menu menu.
+3. Select **Add an Authority**.
+4. Select the **Default Authority** workflow.
+5. Give your new CA a name and subdomain, and change the Authority Type to Advanced Authority.
+6. On the next page, select "Upload external root"
+
+Here, you can upload the PEM file for your existing root CA certificate.
+**Do not upload your root private key to Smallstep.**
+
+## How to Sign the Certificate Signing Request
+
+Smallstep will send a CSR file for you to download and sign.
+
+You will need to transfer the CSR file to your existing root CA and get it signed. Below we have examples of
+how to do this using PEM files, Active Directory Certificate Services, AWS Certificate Manager Private CA, OpenSSL, and CFSSL.
```

Comment:
I find "PEM files" a little out of place in this list. Thoughts on changing this to "step CLI"?

Reply:
Missed Vault in this list